packages/grid/x-data-grid/src/components/toolbar/GridToolbarExport.tsx

```diff
@@ -68,16 +63,32 @@ export const GridPrintExportMenuItem = (props: GridPrintExportMenuItemProps) =>
 
 const GridToolbarExport = React.forwardRef<HTMLButtonElement, GridToolbarExportProps>(
   function GridToolbarExport(props, ref) {
-    const { csvOptions = {}, printOptions = {}, ...other } = props;
+    const { csvOptions = {}, printOptions = {}, excelOptions, ...other } = props;
 
-    if (csvOptions?.disableToolbarButton && printOptions?.disableToolbarButton) {
+    const apiRef = useGridApiContext();
+
+    const defaultButtons = [
+      ...(csvOptions.disableToolbarButton ? [] : [<GridCsvExportMenuItem options={csvOptions} />]),
```

Comment:
Do we intend to move the csv and print options as pre-processor just like for the Excel in the future ?